pkg/fs/dir.go

```diff
@@ -122,7 +181,8 @@ func readDir(scanDir, namePrefix string) (objInfos []ObjectInfo) {
 		// For directories explicitly end with '/'.
 		if isDir {
 			name += "/"
-			size = 0 // Size is set to '0' for directories explicitly.
+			// Size is set to '0' for directories explicitly.
+			size = 0
```

Comment:
Setting the comment followed by 'code' is a practice, you don't need to move it as a heading. Also this is not relevant to this patch.